meson_mesa3d.mk

```diff
@@ -130,27 +141,44 @@ $(SYMLINKS_TARGET): MESA3D_LIB_INSTALL_DIR:=$(dir $(MESON_GEN_FILES_TARGET))/ins
 $(SYMLINKS_TARGET): $(MESON_GEN_FILES_TARGET)
 	# Create Symlinks
 	mkdir -p $$(dir $$@)
-	ls -1 $$(MESA3D_LIB_INSTALL_DIR)/dri/ | PATH=/usr/bin:$$PATH xargs -I{} ln -s -f libgallium_dri.so $$(dir $$@)/{}
-	cp `ls -1 $$(MESA3D_LIB_INSTALL_DIR)/dri/* | head -1` $$(MESA3D_LIB_INSTALL_DIR)/libgallium_dri.so
+	find $$(MESA3D_LIB_INSTALL_DIR)/dri -name \*_dri.so -printf '%f\n' | PATH=/usr/bin:$$PATH xargs -I{} ln -s -f libgallium_dri.so $$(dir $$@)/{}
+	cp `find $$(MESA3D_LIB_INSTALL_DIR)/dri -name \*_dri.so | head -1` $$(MESA3D_LIB_INSTALL_DIR)/libgallium_dri.so
 	touch $$@
```

Comment:
Can we put this

```diff
+    find $$(MESA3D_LIB_INSTALL_DIR)/dri -name \*_drv_video.so -printf '%f\n' | PATH=/usr/bin:$$PATH xargs -I{} ln -s -f libgallium_drv_video.so $$(dir $$@)/{}
+    cp `find $$(MESA3D_LIB_INSTALL_DIR)/dri -name \*_drv_video.so | head -1` $$(MESA3D_LIB_INSTALL_DIR)/libgallium_drv_video.so
```

.. here and drop everything below?

Reply:
But what if someone don't want to build Gallium VA ?

Reply:
Good catch. But it looks like in the current shape of MR we also try to copy them.

Reply:
Also, gallium drivers may be not selected at all. So we need to rework this part to support these cases.

Reply:
ok

Reply:
Please rebase this on top of https://github.com/GloDroid/aospext/pull/13

Reply:
By the way, have you seen this https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/25842 ?

Reply:
> By the way, have you seen this https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/25842 ?

I see, guess I'll try this first then maybe can you help me by bringing this to #13 ? So I can have one less commit.